java/src/jmri/jmrix/loconet/LnPowerManager.java

```diff
@@ -179,8 +180,22 @@ public void run() {
             msg.setOpCode(LnConstants.OPC_RQ_SL_DATA);
             msg.setElement(1, 0);
             msg.setElement(2, 0);
-            tc.sendLocoNetMessage(msg);
+            try {
+                tc.sendLocoNetMessage(msg);
+            } catch (NullPointerException npe) {
+                log.warn("init of LnPowerManager delayed");
+                try {
+                    Thread.sleep(1000); // wait a bit longer for LnPowerManager to init
+                    tc.sendLocoNetMessage(msg);
+                    log.debug("delayed init of LnPowerManager succeeded, sent msg");
+                } catch (InterruptedException e) {
+                    Thread.currentThread().interrupt(); // retain if needed later
```

Comment:
I am not well-versed in JAVA, so I don't know what effect that "interrupting the current thread" would have.